docs/performance/inserts/bulk.rst

```diff
@@ -0,0 +1,419 @@
+again. When the import succeeds, adjust the number of replicas according to
+your `availability`_ requirements.
+
+For example, the ``CREATE TABLE`` statement we used before could be changed
+to the following::
+
+    cr> CREATE TABLE users(
+    ...   id INT primary key,
+    ...   name TEXT,
+    ...   day_joined TIMESTAMP,
+    ...   bio TEXT INDEX using fulltext,
+    ...   address OBJECT (dynamic) AS (
+    ...     city TEXT,
+    ...     country TEXT
+    ...   )
+    ... ) CLUSTERED INTO 12 shards
+    ... WITH (number_of_replicas = 0);
+    CREATE OK, 1 row affected (... sec)
+
+
+.. _bulk-refresh-interval:
+
+Refresh interval
+----------------
+
+Another way to speed up importing is to set the `refresh_interval`_ of the
+table to zero::
+
+    cr> ALTER TABLE users SET (refresh_interval = 0);
+    ALTER OK, -1 rows affected (... sec)
+
+This will disable the periodic `refresh`_ of the table that is needed to
+minimize the effect of eventual consistency and also minimize the overhead
+during import.
+
+.. HIDE:
+
+    cr> DROP TABLE users;
+    DROP OK, 1 row affected (... sec)
+
+You can also set the refresh interval when initially creating the table, like
+so::
+
+    cr> CREATE TABLE users (
+    ...   id INT primary key,
+    ...   name TEXT,
+    ...   day_joined TIMESTAMP,
+    ...   bio TEXT INDEX using fulltext,
+    ...   address OBJECT (dynamic) AS (
+    ...     city TEXT,
+    ...     country TEXT
+    ...   )
+    ... ) CLUSTERED INTO 12 shards
+    ... WITH (
+    ...   number_of_replicas = 0,
+    ...   refresh_interval = 0
+    ... );
+    CREATE OK, 1 row affected (... sec)
+
+When the import has finished, you can set the refresh interval back to a
+reasonable value (milliseconds)::
+
+    cr> ALTER TABLE users SET (refresh_interval = 1000);
+    ALTER OK, -1 rows affected (... sec)
+
+
+.. _bulk-import-data:
+
+Import the data
+===============
+
+Once the table is created, you can start importing the data.
+
+When importing, CrateDB has `native support for JSON data`_, where each line is
+a JSON string representing a single record. Empty lines are skipped. The keys
+of the JSON objects are mapped to columns when imported. Nonexistent columns
+will be created if necessary.
+
+For example, a JSON file might look like this:
+
+.. code-block:: json
+
+    {"id": 1, "name": "foo", "day_joined": 1408312800, "bio": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit.", "address": {"city": "Dornbirn", "country": "Austria"}}
+    {"id": 2, "name": "bar", "day_joined": 1408312800, "bio": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit.", "address": {"city": "Berlin", "country": "Germany"}}
+
+
+Use the `COPY FROM`_ statement to import JSON data directly from a file::
+
+    cr> COPY users FROM '/tmp/best_practice_data/users.json';
+    COPY OK, 150 rows affected (... sec)
+
+Here, CrateDB will check ``/tmp/best_practices_data/`` locally on each node in
```

Comment:
I believe there have to be special considerations when referencing files on Windows file systems. Maybe it would be appropriate to add them below this section as an appropriate `.. TIP` admonition?